src/platform/xbox/platform.c

```diff
@@ -330,13 +335,18 @@ void info_update_callback(lv_timer_t *timer)
         gpu_speed = xbox_get_gpu_frequency();
     }
 
-    xbox_get_temps(&cpu_temp, &mb_temp, &temp_unit);
+    static bool first = 1;
+    if (first)
+    {
+        first = 0;
+        ExQueryNonVolatileSetting(XC_FACTORY_SERIAL_NUMBER, &type, &serial_number, sizeof(serial_number), NULL);
+        ExQueryNonVolatileSetting(XC_FACTORY_ETHERNET_ADDR, &type, &mac_address, sizeof(mac_address), NULL);
+        ExQueryNonVolatileSetting(XC_FACTORY_AV_REGION, &type, &video_region, sizeof(video_region), NULL);
+        ExQueryNonVolatileSetting(XC_FACTORY_GAME_REGION, &type, &game_region, sizeof(game_region), NULL);

```

Comment:
iirc these actually shouldn't be too hard on the system, the kernel caches the eeprom in memory during boot and should be hitting that for `ExQuery`. Still good to avoid wasting time.